packages/premint-sdk/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./premint-client";
+
+export * from "./preminter";
+
+export * as ZoraPremintApiClient from "./premint-api-client";
```

Comment:
with the export structured this way, would you need to do:

```ts
import { ZoraPremintApiClient } from "./premint-api-client";

ZoraPremintApiClient.PremintApiClient
```

why not just `export * from './premint-api-client'`

Reply:
Yep good idea.